packages/react/src/components/Popup/Popup.tsx

```diff
@@ -211,6 +220,9 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
       e.preventDefault()
       this.setPopupOpen(true, e)
     },
+    preventScroll: e => {
```

Comment:
```suggestion
    closeOnScroll: e => {
```

Should have a different name now...